templates/base.html

```diff
@@ -8,7 +8,7 @@
     <link rel="preload" as="style" href="dark.css" />
     <link rel="preload" as="style" href="light.css" />
     <link rel="stylesheet" href="{{ base_path.join(meta.highlight.style.name) }}">
-    <link rel="icon" href="{{ base_path.join("favicon.png") }}" type="image/png">
+    <link rel="icon" href="{{ base_path.join("favicon.ico") }}" type="image/png">
```

Comment:
But … it's a PNG?!

Reply:
There is still the `type` hint and browsers can still figure out the image type out of the binary file header.

Reply:
I mean, we do not serve the asset file, aren't we?

Reply:
I'm stupid, sorry :-)